src/ledger/AccountFrame.h

```diff
@@ -23,6 +23,9 @@ namespace stellar
 class LedgerManager;
 class LedgerRange;
 
+int64_t getBuyingLiabilities(AccountEntry const& acc, LedgerManager const& lm);
```

Comment:
Looks like versions of those for AccountEntry and TrustLineEntry could be merged as a template.